src/countdown.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  Text,
+  Image,
+  View,
+  Button
+} from 'react-native';
+
+export default class CountDown extends Component {
+  constructor() {
+    super();
+    this.state = {
+      time: {},
+      seconds: 180,
+    };
+    this.timer = 0;
+    this.startTimer = this.startTimer.bind(this);
+    this.countDown = this.countDown.bind(this);
+  }
+  secondsToTime(secs){
+    let minutes = Math.floor(secs / 60);
+
+    let divisor_for_seconds = secs % 60;
+    let seconds = Math.ceil(divisor_for_seconds);
+
+    let obj = {
+      m: minutes,
+      s: seconds
+    };
+    return obj;
+  }
+  componentDidMount() {
+    let timeLeftVar = this.secondsToTime(this.state.seconds);
+    this.setState({ time: timeLeftVar });
+  }
+  startTimer() {
+    if (this.timer == 0) {
+      this.timer = setInterval(this.countDown, 1000);
+    }
+  }
+  countDown() {
+    let seconds = this.state.seconds - 1;
+    this.setState({
+      time: this.secondsToTime(seconds),
+      seconds: seconds,
+    });
+    if (seconds == 0) {
+      clearInterval(this.timer);
+    }
+  }
+  render() {
+    if(this.state.time.s < 10)
```

Comment:
In order to keep the rendering logic simple,  I think we should move this logic in `secondsToTime` function,
and this function should only return (<Text>this.secondsToTime()</Text>)